host-controller/src/main/java/org/jboss/as/domain/controller/transformers/KernelAPIVersion.java

```diff
@@ -28,8 +28,12 @@
  *
  * @author Brian Stansberry
  */
-enum KernelAPIVersion {
+public enum KernelAPIVersion {
 
+    // EAP 6.2.0
+    VERSION_1_5(1, 5, 0),
+    // EAP 6.3.0
+    VERSION_1_6(1, 6, 0),
```

Comment:
This is fine but someday as part of an overall cleanup of code related to EAP < 6.4 in mixed domain we can clean this out too.

Now the EAP domain.xml actually includes host-exclude resources for 6.2 and 6.3, but we don't actually support that.